src/compute/src/render/mod.rs

```diff
@@ -271,67 +273,59 @@ pub fn build_compute_dataflow<A: Allocate>(
         // If there exists a recursive expression, we'll need to use a non-region scope,
         // in order to support additional timestamp coordinates for iteration.
         if recursive {
-            scope
-                .clone()
-                .iterative::<PointStamp<usize>, _, _>(|region| {
-                    let mut context =
-                        crate::render::context::Context::for_dataflow_in(&dataflow, region.clone());
-
-                    for (id, (oks, errs)) in imported_sources.into_iter() {
-                        let bundle = crate::render::CollectionBundle::from_collections(
-                            oks.enter(region),
-                            errs.enter(region),
-                        );
-                        // Associate collection bundle with the source identifier.
-                        context.insert_id(id, bundle);
-                    }
+            scope.clone().iterative::<PointStamp<u64>, _, _>(|region| {
+                let mut context =
+                    crate::render::context::Context::for_dataflow_in(&dataflow, region.clone());
 
-                    // Import declared indexes into the rendering context.
-                    for (idx_id, idx) in &dataflow.index_imports {
-                        let export_ids = dataflow.export_ids().collect();
-                        context.import_index(
-                            compute_state,
-                            &mut tokens,
-                            export_ids,
-                            *idx_id,
-                            &idx.0,
-                        );
-                    }
+                for (id, (oks, errs)) in imported_sources.into_iter() {
+                    let bundle = crate::render::CollectionBundle::from_collections(
```

Comment:
This looks like an interesting block to test

Reply:
This is actually already covered by `with_mutually_recursive.slt`.

Reply:
Thanks, you are right, this was an issue in my coverage implementation, fixed with https://github.com/MaterializeInc/materialize/pull/19193